### PR TITLE
change avatar border radius to 50%

### PR DESCRIPTION
**Pull request by ajb**
pay special attention to the `.drop_master` stuff when reviewing...

### CODE REVIEW

**Comment by ajb** [comment]
cf https://github.com/dobtco/screendoor-v2/issues/1861

When merging, we should quickly bump `dvl-core` on all apps that use the navbar, where this will be a pretty prominent change.

File: vendor/assets/stylesheets/dvl/core/media.scss [hunk 1]
```diff
@@ -21,3 +21,9 @@ img, video {
   width: auto\9;
   height: auto !important;
 }
+
+// Enforcing component consistency
+
+@mixin avatar {
+  border-radius: 50%;
+}
```
Comment (ajb) [inline vendor/assets/stylesheets/dvl/core/media.scss]: @jrubenoff 

img

lol

Comment (jrubenoff) [inline vendor/assets/stylesheets/dvl/core/media.scss]: Ah, thank you. Didn't have `automatic_white_space`.

Comment (ajb) [inline vendor/assets/stylesheets/dvl/core/media.scss]: Ah shit, I missed the most important one: `"ensure_newline_at_eof_on_save": true`

**Pull request merged**